Source/Core/Core/Core.cpp

```diff
@@ -750,7 +751,7 @@ State GetState()
   if (s_hardware_initialized)
   {
     auto& system = Core::System::GetInstance();
-    if (system.GetCPU().IsStepping() || s_frame_step)
```

Comment:
Wonder if this change should be in its own commit with a description?  It wasn't obvious to me why this changed until I looked at the issue comment.

Reply:
Yeah I could split them. Removing that is the crash fix and the rest is to re-implement the intended fix of having that there.